workloads/api-load/run.sh

```diff
@@ -9,6 +9,8 @@ log "###############################################"
 log "api-load tests: ${TESTS}"
 log "###############################################"
 
+prepare_tests
```

Comment:
@vishnuchalla are these intended ? I think you have to rebase your code and update this PR.

Reply:
okay, that make sense. you moved it out of the function.